examples/sql/read_json.py

```diff
@@ -0,0 +1,156 @@
+# /// script
+# requires-python = ">=3.10"
+# dependencies = [
+#     "duckdb==1.1.1",
+#     "marimo",
+#     "pandas==2.2.3",
+#     "pyarrow==17.0.0",
+#     "vega-datasets==0.9.0",
+# ]
+# ///
+
+import marimo
+
+__generated_with = "0.9.1"
+app = marimo.App(width="medium")
+
+
+@app.cell(hide_code=True)
+def __(mo):
+    mo.md(
+        """
+        # Read JSON
+
+        This notebook shows how to read a JSON file from a local file or a URL into an in-memory table.
+        """
+    )
+    return
+
+
+@app.cell(hide_code=True)
+def __():
+    import marimo as mo
+    import pandas as pd
```

Comment:
super nit; but i feel like we should use polars (instead of pandas) when possible

Reply:
yea i agree in general but i'd like to host these on WASM too

Reply:
oh yes good call - we did discuss that